x-pack/plugins/actions/server/usage/actions_telemetry.ts

```diff
@@ -145,7 +162,28 @@ export async function getInUseTotalCount(callCluster: LegacyAPICaller, kibanaInd
     },
   });
 
-  return actionResults.aggregations.refs.actionRefIds.value.total;
+  const bulkFilter = Object.entries(
+    actionResults.aggregations.refs.actionRefIds.value.connectorIds
+  ).map(([key]) => ({
+    id: key,
+    type: 'action',
+    fields: ['id', 'actionTypeId'],
+  }));
+  const actions = await actionsBulkGet(bulkFilter);
+  const countByType = actions.saved_objects.reduce(
+    (actionTypeCount: Record<string, number>, action) => {
+      const hasFirstSymbolDot = action.attributes.actionTypeId.startsWith('.');
```

Comment:
Since this code is the same as on line 65, I think we should reorganize it into a single function in this module, that's called in both places.

Reply:
The reason why we need this replacement of the first symbol '.' is from this [issue](https://github.com/elastic/telemetry/issues/283) 
```
"reason" : "object field starting or ending with a [.] makes object resolution ambiguous: [.webhook]"
"reason" : "object field starting or ending with a [.] makes object resolution ambiguous: [.index-threshold]"
"reason" : "object field starting or ending with a [.] makes object resolution ambiguous: [.server-log]"
```
and commented [here](https://github.com/elastic/telemetry/issues/283#issuecomment-604166195). I agree that we should have a single method for doing this.

Reply:
I think that we should add handling for end symbol '.' as well.

Reply:
If there are no first symbol '.', easier to keep the original alertTypeId and actionTypeId which are easier to identify. So I think we should keep only replacements which are required.

Reply:
cool, thx.  

I'd guess we won't see any types that end with `.`, but I guess it could cause problems if we are doing "text/keyword" multi-fields (eg, where you would append `.keyword` to the field), so probably makes sense to process a trailing `.` as well.  Just in case :-)